docker-compose/config/pm2-dev.json

```diff
@@ -3,8 +3,8 @@
     "name"         : "KuzzleServer",
     "script"       : "bin/kuzzle",
     "args"         : ["start"],
-    "watch"        : ["lib", "config", "bin", "default.config.js"],
-    "ignore_watch" : ["node_modules"],
-    "node_args": "--inspect"
+    "watch"        : ["lib", "config", "bin", "default.config.js", "plugins/enabled"],
```

Comment:
not sure that we only want to watch `.js` files. Like, imagine a plugin that loads, for whatever reason, a `.json` config file -or another format (e.g. `.yml`) for parsing, it wouldn't be watched.
I know it doesn't really make sense, but who knows? Future use-cases are always mysterious (and sometimes completely stupid) but I'm not sure it's worth restraining the scope for the sake of minimalism.